ms-windows/templates/3.1/x64/openssl.iss

```diff
@@ -0,0 +1,152 @@
+; Script generated by the Inno Setup Script Wizard.
+; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
+
+[Setup]
+AppName=OpenSSL (64-bit)
+AppVersion=[[DOTTED_VERSION]]
+AppVerName=OpenSSL [[DOTTED_VERSION]] (64-bit)
+AppPublisher=OpenSSL Installer Team
+AppPublisherURL=https://www.openssl.org
+AppSupportURL=https://github.com/openssl/installer
+AppUpdatesURL=https://www.openssl.org
+VersionInfoVersion=[[WINDOWS_DOTTED_VERSION]]
+DefaultDirName={pf}\OpenSSL
+DisableDirPage=no
+DefaultGroupName=OpenSSL
+SourceDir=[[INSTALLER_SOURCE_DIR]]
+OutputBaseFilename=OpenSSL-[[DOTTED_VERSION]]-x64
+LicenseFile=license.txt
+; SetupIconFile=compiler:SetupClassicIcon.ico
+PrivilegesRequired=admin
+Compression=bzip
+ArchitecturesAllowed=x64
+ArchitecturesInstallIn64BitMode=x64
+; Minimum version supported is Windows XP.

```

Comment:
I do wonder whether we can bump this up for 3.1 (maybe even 3.0). There are certain optimisations and benefits that we get if we compile OpenSSL targeting a more recent version of Windows. People that want XP support can still build from source.

Reply:
If this is the route to take, Inno Setup 5 can be dropped altogether.  When setting up the Inno Setup 5 dependency, it triggers the UAC prompt even though it ends up doing nothing requiring elevation.  Dropping that dependency cleans up the initialization phase.  Inno Setup 5 usage exists for XP support.  So if we don't need that, the dependency can be dropped and the `info.json` files and the installers adjusted accordingly for Inno Setup 6.

Let me know what OSes to support for which versions.